src/core/model_concrete.jl

```diff
@@ -43,19 +43,24 @@ single container. Offers the variants:
 """
 function SingleContainerABM(
     ::Type{A},
-    agentstep!::G,
-    modelstep!::K,
     space::S = nothing;
+    agent_step!::G = dummystep,
+    model_step!::K = dummystep,
     container::Type = Dict{Int},
     scheduler::F = Schedulers.fastest,
     properties::P = nothing,
     rng::R = Random.default_rng(),
     warn = true
 ) where {A<:AbstractAgent,S<:SpaceType,G,K,F,P,R<:AbstractRNG}
+    if agent_step! == dummystep && model_step! == dummystep
+        @warn "From version 6.0 it is necessary to pass agent_step! and model_step! when defining 
```

Comment:
It's not necessary to pass both. Suggestion: "From version 6.0 it is necessary to pass at least one of agent_step! or model_step! when defining"